.eslintrc.json

```diff
@@ -4,8 +4,19 @@
     "es6": true,
     "jest": true
   },
-  "extends": [
-    "@mapbox/eslint-config-mapbox",
-    "prettier"
+  "extends": ["@mapbox/eslint-config-mapbox", "prettier"],
+  "overrides": [
+    {
+      "files": ["*.test.js", "*.spec.js"],
+      "rules": {
+        "node/no-unsupported-features": [
+          "error",
+          {
+            "version": 8,
```

Comment:
I am thinking of using node v8 to ease testing, this shouldn't affect the user using underreact I guess.

Reply:
I think we should probably test in the version of Node that users will have installed. If we write the tests for Node 8, we can't run them in CI on Node 6.

Reply:
Should we change this before merging, @kepta?